src/main/java/frc/robot/commands/IntakingNoteCommand.java

```diff
@@ -0,0 +1,44 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.ParallelRaceGroup;
+import edu.wpi.first.wpilibj2.command.RunCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.subsystems.ConveyorSubsystem;
+import frc.robot.subsystems.ElevatorSubsystem;
+import frc.robot.subsystems.FloorIntakeSubsystem;
+import frc.robot.subsystems.FlyWheelSubsystem;
+import frc.robot.subsystems.PivotSubsystem;
+import frc.robot.commands.ConveyorCommands.ConveyLineBreak;
+import frc.robot.commands.FloorIntakeCommands.FI_LinebreakCommand;
+import frc.robot.commands.ShootingPosCommands.BasePosition;
+
+public class IntakingNoteCommand extends SequentialCommandGroup{
+    FloorIntakeSubsystem floorIntake;
+    ConveyorSubsystem conveyor;
+    ElevatorSubsystem elevator;
+    PivotSubsystem pivot;
+    FlyWheelSubsystem flyWheel;
+
+    public IntakingNoteCommand(FloorIntakeSubsystem floorIntake, ConveyorSubsystem conveyor,
+        ElevatorSubsystem elevator, PivotSubsystem pivot, FlyWheelSubsystem flywheel){
+        
+        this.floorIntake = floorIntake;
+        this.conveyor = conveyor;
+        this.elevator = elevator;
+        this.pivot = pivot;
+        this.flyWheel = flywheel;
```

Comment:
Note that these values are only being used inside the constructor, so these don't need to be assigned to instance variables.

Consider if you deleted these lines and lines 18-22. The code would still work the same.

Reply:
Got it :)

Reply:
lines 27-31 were mad when I did that, so I just deleted those and the code seems happy with just the objects in the parameters.